spaces/Sogur/index.json

```
@@ -0,0 +1,28 @@
+{
+  "key": "Sogur",
```

The key will be the url and we want it all in lowercase. Please modify the value of the key by something like `sogur`. Thank you.
```suggestion
  "key": "sogur",
```

---

The space directory should be in lowercase too. `/spaces/sogur`

---

done